pkgs/applications/editors/xmlcopyeditor/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, aspell, boost, expat, expect, intltool, libxml2, libxslt, pcre, wxGTK, xercesc }:
+
+stdenv.mkDerivation rec {
+  name = "xmlcopyeditor-${version}";
+  version = "1.2.1.3";
```

Comment:
1.2.1.5 is available

Reply:
It's a WIP macOS (only?) port as I understand it.

http://xml-copy-editor.sourceforge.net/ and https://sourceforge.net/projects/xml-copy-editor/files/xmlcopyeditor-linux/ still points to v1.2.1.3 for Linux (or Windows).

Reply:
The second link is not the front website but the list of source archives for Linux.
https://repology.org/metapackage/xmlcopyeditor/versions
No one ships v1.2.1.5.

So I'm going to stick to the version intended by the original author of this PR who's going to be its maintainer. @candeira should open a new PR later once upstream got its release process straighten out.